### DISCUSSION TITLE
Directly Deploying Verifier to Cyan not working - Testnet not working either

### DISCUSSION THREAD

**Discussion by aignermax** [Web3Auth/Web3Auth#716] [Help]
Hi there, 
I am currently trying to create a verifier on the web3Auth dashboard. Unfortunately I get the error message that 
"You are required to first have a testnet with the same project name in your dashboard". 
On the same prompt I also get the message "Please deploy directly to Cyan if its taking too long or if you're experiencing issues". 
As the Ropsten testnet is currently getting merged, I cannot create a Verifier on the testnet. so I am kind of stuck.. or am I missing something?

![image](https://user-images.githubusercontent.com/18228325/195348780-d19f64a1-e033-400f-86dc-5ee8a028ea5a.png)

**Comment by aignermax** [comment]
Actually now the Ropsten testnet is fast enough now, but the message is a bit misleading still, once the testnet does not work anymore. Anyway waiting for an hour fixed the problem for me.

**Comment by rinzler99** [comment]
Marking anwered as issue is already fixed.
